### PR TITLE
Updated table with read_fwf()

### LINKED ISSUE #27884: DOC: read_fwf not appears on IO Formats table as function
#### Problem description
I Think that needs to appear on that table because is not clearly documented

image

### PR DESCRIPTION
**Pull request by isantolin**
- [X] closes #27884 
- [x] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

@TomAugspurger Can you merge?

### CODE REVIEW

File: doc/source/user_guide/io.rst [hunk 1]
```diff
@@ -28,6 +28,7 @@ The pandas I/O API is a set of top level ``reader`` functions accessed like
     :delim: ;
 
     text;`CSV <https://en.wikipedia.org/wiki/Comma-separated_values>`__;:ref:`read_csv<io.read_csv_table>`;:ref:`to_csv<io.store_in_csv>`
+    text;`TXT <https://www.oracle.com/webfolder/technetwork/data-quality/edqhelp/Content/introduction/getting_started/configuring_fixed_width_text_file_formats.htm>`__;:ref:`read_fwf<io.fwf_reader>`
```
Comment (jreback) [inline doc/source/user_guide/io.rst]: i might say fixed with text but not sure will fit

Comment (isantolin) [reply doc/source/user_guide/io.rst]: @TomAugspurger Can you resolve?

Comment (TomAugspurger) [reply doc/source/user_guide/io.rst]: I'm not sure what you mean.

Comment (isantolin) [reply doc/source/user_guide/io.rst]: @TomAugspurger He´s talking about "Format Type" on "IO Tools" table... I think that "Text" is Correct in read_fwf() case

Comment (TomAugspurger) [reply doc/source/user_guide/io.rst]: OK. Sounds fine.

**Comment by isantolin** [comment]
@TomAugspurger Can you merge, or need any other step?

**Comment by TomAugspurger** [comment]
Thanks.

**Pull request merged**